code/src/main/webapp/CSS/style.css

```diff
@@ -1,13 +1,57 @@
-h1 {  
-  color: rgba(43, 141, 190, 1);
-  text-align: center;
+body, html {
+  height: 100%;
+  margin: 0;
+  color: #fefff2;
 }
-p {
-  color: rgba(43, 142, 190, 1);
-  text-align: center;
+
+h1, p, a {
+  margin: 0;
 }
-a {
-  color: rgba(43, 142, 190, 1);
+
+div {
+ /* center text */
+  position: fixed;
+  width: 100%;
   text-align: center;
 }
 
+#wrapper {
```

Comment:
(Here and elsewhere) please use more verbose and descriptive IDs. background-image-wrapper might be better here.

Reply:
done